sdks/go/examples/xlang/cogroup_by/cogroup_by.go

```diff
@@ -76,6 +77,13 @@ func init() {
 	beam.RegisterFunction(formatFn)
 	beam.RegisterFunction(getKV)
 	beam.RegisterFunction(sumCounts)
+
```

Comment:
We can cut the above calls

Reply:
Whoops, forgot to get them all out. Thanks!